### ISSUE TITLE
Announce status changes

### ISSUE THREAD

**Issue by WestphalDenn** [bigbluebutton/bigbluebutton#11372]
I am writing this as a blind user. Getting this behaviour in would greatly improve the user experience using a screen reader. 

For screenreader users it is very time consuming to get an overview if someone changed their status. That comes into play when raised hands are used so that people aren't talking over one another. 

Current behaviour: 
Screenreader users have to always arrow up and down through the list of participants to hear every status set individually which creates a lot of talking by the screenreader 

Expected behaviour
When someone changes their status, the screenreader should announce who changed their status and what the new status is. Perhaps something like "X changed their status from y to z". 

That change would effect all browsers and all operating systems as well as all screen readers.
